vault/identity_store_oidc.go

```diff
@@ -585,17 +585,7 @@ func (i *IdentityStore) pathOIDCGenerateToken(ctx context.Context, req *logical.
 	}
 
 	// generate an OIDC token from entity data
-	accessorEntry, err := i.core.tokenStore.lookupByAccessor(ctx, req.ClientTokenAccessor, false, false)
-	if err != nil {
-		return nil, err
-	}
-
-	te, err := i.core.LookupToken(ctx, accessorEntry.TokenID)
-	if err != nil {
-		return nil, err
-	}
-
-	if te == nil || te.EntityID == "" {
+	if req.EntityID == "" {
```

Comment:
@kalafut could we just use req.EntityID here instead?

Reply:
yep. changed.